src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Drawable, Randomizable {
```

Comment:
According to the checklist, you should not use abstract classes to set behavior for classes. Consider using interfaces or regular classes depending on the need for shared state or behavior.